src/common/utilities/WorkspaceInfoFinderUtil.ts

```diff
@@ -36,3 +39,24 @@ export function getPortalsOrgURLs(workspaceRootFolders: WorkspaceFolder[] | null
     }
     return output;
 }
+
+export function getWebsiteRecordID(workspaceFolders: { uri: string }[], telemetry: ITelemetry): string {
+    try {
+        if (!workspaceFolders || workspaceFolders.length === 0) {
+            return "";
+        }
+
+        const workspaceRootFolder = workspaceFolders[0];
+        const websiteYmlPath = path.join(workspaceRootFolder.uri, 'website.yml');
+        if (fs.existsSync(websiteYmlPath)) {
+            const fileContent = fs.readFileSync(websiteYmlPath, 'utf8');
+            const parsedYaml = parse(fileContent);
+            if (parsedYaml && parsedYaml.adx_websiteid) {
+                return parsedYaml.adx_websiteid;
+            }
+        }
+    } catch (exception) {
+        sendTelemetryEvent(telemetry, { methodName: getWebsiteRecordID.name, eventName: 'getWebsiteRecordID', exception: exception as Error });
```

Comment:
Can we move this string to constant file?

Reply:
done